framework/src/main/java/org/checkerframework/framework/util/AnnotationMirrorSet.java

```diff
@@ -133,4 +133,16 @@ public boolean removeAll(Collection<?> c) {
     public void clear() {
         shadowSet.clear();
     }
+
+    /**
+     * Returns a new {@link AnnotationMirrorSet} that contains {@code value}.
+     *
+     * @param value AnnotationMirror to put in the set
+     * @return a new {@link AnnotationMirrorSet} that contains {@code value}.
+     */
+    public static AnnotationMirrorSet singleton(AnnotationMirror value) {
```

Comment:
Wrong name.  Look at Set implementation.

Reply:
This method has the same name as [Collections#singleton](
https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#singleton(T)).  So, I think it is the right name.  We could also call it `signletonSet`.

Reply:
That method states: `Returns an immutable set containing only the specified object.`
So should this method also return something immutable?
Or the javadoc should highlight this difference to the standard API behavior.

Reply:
Good point.  I changed it to `singleElement`.